tests/integration/pilot/ingress_test.go

```diff
@@ -257,5 +260,23 @@ spec:
 					apps.Ingress.CallEchoWithRetryOrFail(ctx, c.call, retry.Timeout(time.Minute*2))
 				})
 			}
+
+			t.Run("status", func(t *testing.T) {
+				if !ctx.Environment().(*kube.Environment).Settings().LoadBalancerSupported {
```

Comment:
noticed that prow runs pilot tests w/ the `--istio.test.loadbalancer=false`, so where does this test get run?

Reply:
Ah I did not know that, I thought we were deploying metallb (allows LB on kind) for everything but looks like just for multicluster. We do have MC tests that run these which would trigger it I believe, although I think they are still WIP and hidden currently